CMakeLists.txt

```diff
@@ -273,6 +294,64 @@ if(USEGPU)
 	endif()
 endif()
 
+ 
+if(USEARROW)
+
+	#list(APPEND Nyxus_LIBRARIES ${ARROW_LIB})
+	list(APPEND Nyxus_LIBRARIES arrow_shared)
+
+
+	if(PARQUET_LINK_SHARED)
+		list(APPEND Nyxus_LIBRARIES parquet_shared)
+	else()
+		list(APPEND Nyxus_LIBRARIES parquet_static)
+	endif()
+
+	if (BUILD_LIB)
+		# Find Python
+		find_package(Python REQUIRED COMPONENTS Interpreter Development)
```

Comment:
can you check if we can get away w/o using this? Since we already find Python at the beginning.